### DISCUSSION TITLE
NetData for monitoring

### DISCUSSION THREAD

**Discussion by samuelbohl** [coollabsio/coolify#1834] [Service Template Requests]
At the moment, I am flying blind in terms of monitoring my services. It would be great if we could integrate a monitoring solution like NetData to get a better picture as to how the resources are used.

**Comment by kerbymart** [comment]
Hi @samuelbohl I suppose you can easily install Netdata outside of Coolify. Then just use SSH proxy to access the default port 19999 locally.
